File: Python/1012.py
val = input().split()
a = float(val[0])
b = float(val[1])
c = float(val[2])

area = a*c/2
raio = 3.14159 * (c**2)
trap = (a+b)*c/2
quad = b*b
ret = a*b

print(f'TRIANGULO: {area:.3f}')
print(f'CIRCULO: {raio:.3f}')
print(f'TRAPEZIO: {trap:.3f}')
print(f'QUADRADO: {quad:.3f}')
print(f'RETANGULO: {ret:.3f}')

File: C/1012.c
#include <stdio.h>
 
int main() {
 
    float a,b,c, total;
    scanf("%f%f%f", &a,&b,&c);
    total = a*c;
    printf("TRIANGULO: %.3f\n", total/2);
    total = c*c;
    printf("CIRCULO: %.3f\n", total*3.14159);
    total = a+b;
    printf("TRAPEZIO: %.3f\n", total*c/2);
    total = b*b;
    printf("QUADRADO: %.3f\n", total);
    total = a*b;
    printf("RETANGULO: %.3f\n", total);
}
